src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -106,14 +106,22 @@ void Zone::SizeWindowToZone(HWND window, HWND zoneWindow) noexcept
     }
 
     WINDOWPLACEMENT placement{};
-    ::GetWindowPlacement(window, &placement);
-    placement.rcNormalPosition = zoneRect;
-    placement.flags |= WPF_ASYNCWINDOWPLACEMENT;
+    
```

Comment:
`::GetWindowPlacement(window, &placement);` should be called before `if ((placement.showCmd & SW_SHOWMINIMIZED) == 0)`

Reply:
Of course! 😱 My mistake.